internal/precache/precache-workload/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.hub.docker.com/library/golang:1.20 as builder
```

Comment:
Please roll this up into the LCA image as a secondary binary with a single vendor folder, rather than having a separate image, as has been done with the ibu-imager.

Reply:
Done